prime-router/src/main/kotlin/SenderConfiguration.kt

```diff
@@ -0,0 +1,10 @@
+package gov.cdc.prime.router
```

Comment:
Very minor note:  for similar "sub-objects" in Receiver, the data class is right inside Receiver, not in a separate class.

Reply:
Good point. I was looking at it like the TranslatorConfiguration class which can be overloaded and lives separately. I was thinking there might be a chance we ended up needing to subclass this in the future, but I could be overthinking it.